invio/docker-compose.yml

```
@@ -0,0 +1,35 @@
+services:
+
+  app_proxy:
+    environment:
+      APP_HOST: invio_frontend_1
+      APP_PORT: 8000
+      PROXY_AUTH_ADD: "false"
+
+  backend:
+    image: ghcr.io/kittendevv/invio-backend:v1.7.0@sha256:1d7bc45a698e6ee753e34bbb8e7d1d56c8afc338a6cca8f025894667130555ed
+    user: "1000:1000"
+    environment:
+      ADMIN_USER: "admin"
+      ADMIN_PASS: ${APP_PASSWORD}
+      JWT_SECRET: ${APP_SEED}
+      SESSION_TTL_SECONDS: 43200
+      SECURE_HEADERS_DISABLED: "false"
+      ENABLE_HSTS: "true"
+      DATABASE_PATH: ${APP_DATA_DIR}/data/db:/app/data/invio.db
```

Sorry, my bad I thought it was for the external database, but it’s actually for the internal container database. So, we can change it to the container path instead of using a bind mount.
```suggestion
      DATABASE_PATH: /app/data/invio.db
```